src/routes/DigitalAssets/routes/List/AssetsContainer.js

```diff
@@ -0,0 +1,25 @@
+// @flow
+
+import { connect } from 'react-redux'
+
+import { DigitalAssetsGrid } from 'components'
```

Comment:
need to import:
`route name of parent` + `route name` + `View` = `DigitalAssetsListView`

And the container file name should be `view name` + `Container` = `DigitalAssetsListViewContainer`